scripts/ci_check_no_file_changes.sh

```diff
@@ -44,11 +39,25 @@ do
     echo "Invalid check: \"${CHECK}\". Falling back to exiting with FAILURE code"
     exit 1
   fi
+  REGEXES=(${REGEXES[@]} ${REGEX})
```

Comment:
nit: shellcheck complains about:
```
  REGEXES=(${REGEXES[@]} ${REGEX})
           ^-----------^ SC2206: Quote to prevent word splitting/globbing, or split robustly with mapfile or read -a.
                         ^------^ SC2206: Quote to prevent word splitting/globbing, or split robustly with mapfile or read -a.
```